src/components/quickStarts.stories.mdx

```diff
@@ -112,7 +48,13 @@ const MyApp = () => {
 };
 ```
 
-_Note_ - The view-only mode doesn't impact the ability to view content or listen to audio in your meeting experience.
+> **Note:** In some cases, joining the meeting in view-only mode will result in attendees not being able to join the meeting successfully. A typical feature of this issue is that when an attendee joins the meeting, the roster is empty and the attendee cannot receive audio and video feeds from others.
```

Comment:
Can we clarify which cases or link to existing issues if any?
"Joining a meeting in view-only mode may result in attendees not able to join that meeting successfully. This results into an issue where the roster remains empty and the attendee cannot receive audio and video feeds from others."